README.md

```diff
@@ -314,6 +314,9 @@ All methods available are:
 - "AIS_Kriging_S"
 - "SMCS"
 - "WR_SMC"
+- "Federated SBS linear"
+- "Federated SBS quadratic",
+- "Federated SBS constant"
```

Comment:
The README.md changed a lot, now this section belong to the documentation.md file.
Can you add it in the documentation.md ? 
-> mplc/documentation.md